libs/utils/android/workloads/youtube.py

```diff
@@ -42,31 +40,46 @@ def __init__(self, test_env):
         # Set of output data reported by Jankbench
         self.db_file = None
 
-    def run(self, out_dir, collect,
-            video_url, video_duration_s):
+    def run(self, out_dir, video_url, video_duration_s, collect=''):
+        """
+        Run single YouTube workload.
+
+        :param out_dir: Path to experiment directory where to store results.
+        :type exp_dir: str
+
+        :param video_url: Video URL to be played
+        :type video_url: str
+
+        :param duration_s: Play video for this required number of seconds
+        :type duration_s: int
+
+        :param collect: Specifies what to collect. Possible values:
+            - 'energy'
+        :type collect: list(str)
+        """
 
         # Keep track of mandatory parameters
         self.out_dir = out_dir
         self.collect = collect
 
         # Unlock device screen (assume no password required)
-        System.menu(self.target)
+        System.menu(self._target)
         # Press Back button to be sure we run the video from the start
-        System.back(self.target)
+        System.back(self._target)
 
         # Use the monkey tool to start YouTube without playing any video.
         # This allows to subsequently set the screen orientation to LANDSCAPE
         # and to reset the frame statistics.
-        System.monkey(self.target, self.package)
+        System.monkey(self._target, self.package)
 
         # Force screen in LANDSCAPE mode
-        Screen.set_orientation(self.target, portrait=False)
+        Screen.set_orientation(self._target, portrait=False)
```

Comment:
It's done now! I posted the comment a short while before I got the chance to push the branch. You move fast!